pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -0,0 +1,195 @@
+#
+# Factory for creation of a population based on an input file
+#
+
+import numpy as np
+import pandas as pd
+import random
+import copy
+
+from pyEpiabm.core import Household, Population, Cell
+from pyEpiabm.core.microcell import Microcell
+from pyEpiabm.property import InfectionStatus
+
+
+class FilePopulationFactory:
+    """ Class that creates a population  based on an input .csv file.
+    """
+    @staticmethod
+    def make_pop(input_file: str, random_seed: int = None):
+        """Initialize a population object from an input csv file, with one
+        row per microcell. A uniform multinomial distribution is
+        used to distribute the number of people into the different households
+        within each microcell. A random seed may be specified for reproducible
+        populations.
+
+        Input file contains columns:
+            * `cell`: ID code for cell (Uses hash if unspecified)
```

Comment:
Good spot, I had missed this! I put this in originally because the default id for a cell is the hash if the cell ID is not specified. However, the file inputter would not work then, because it wouldn't know which microcell is in each cell, hence it will throw an error if the cell id is not specified. I have updated the docs to reflect this